DuckDuckGo/BookmarksViewController.swift

```diff
@@ -21,26 +21,29 @@ import UIKit
 import Core
 import os.log
 
+// swiftlint:disable file_length
+
 class BookmarksViewController: UITableViewController {
 
+    @IBOutlet weak var addFolderButton: UIBarButtonItem!
     @IBOutlet weak var editButton: UIBarButtonItem!
```

Comment:
We probably should disable Add Folder & Edit buttons when in search mode (keyboard can be invisible when using external one).